src/content/concepts/module-federation.mdx

```diff
@@ -25,6 +24,8 @@ Multiple separate builds should form a single application. These separate builds
 
 This is often known as Micro-Frontends, but is not limited to that.
 
+<StackBlitzPreview example="module-federation?terminal=" />
```

Comment:
Same problem as another one.

Reply:
> Btw, it would be great to avoid force pushing :) Force pushing makes the review process difficult as we have to look over all the changes every time.

I am sorry for inconvenience. I will keep this in mind in my future PRs. Thank you for the suggestions!.